packages/logs-syslog/logs-syslog.0.3.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "Hannes Mehnert <hannes@mehnert.org>"
+authors: ["Hannes Mehnert <hannes@mehnert.org>"]
+homepage: "https://github.com/hannesm/logs-syslog"
+doc: "https://hannesm.github.io/logs-syslog/doc"
+dev-repo: "git+https://github.com/hannesm/logs-syslog.git"
+bug-reports: "https://github.com/hannesm/logs-syslog/issues"
+license: "ISC"
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.1.0" & build}
```

Comment:
```suggestion
  "dune" {>= "1.1.0"}
```

Reply:
Any chance you could commit this fix upstream? Shouldnt have a `{build}` annotation on dune dependencies in opam.

Reply:
oh sure, sorry about this. done in 3988a4b.